cmd/containerd-shim-runhcs-v1/pod.go

```diff
@@ -186,7 +192,14 @@ func createPod(ctx context.Context, events publisher, req *task.CreateTaskReques
 				Pid:        0,
 			})
 	} else {
-		// LCOW requires a real task for the sandbox
+		if isWCOW {
+			// The pause container activation will immediately exit on Windows
+			// because there is no command. We forcibly update the command here
+			// to keep it alive.
+			s.Process.CommandLine = "cmd /c ping -t 127.0.0.1 > nul"
```

Comment:
This will consume resources. What about `cmd /c pause`? (Not sure if that will work due to tty). Or even `timeout /t -1`?

Reply:
Does pause exist?!!

Reply:
Ok so this doesnt work. Leaving as is.